fly/commands/internal/flaghelpers/team_flag.go

```diff
@@ -35,3 +36,16 @@ func (flag *TeamFlag) Complete(match string) []flags.Completion {
 func (flag TeamFlag) Name() string {
 	return string(flag)
 }
+
+func (flag TeamFlag) LoadTeam(target rc.Target) (concourse.Team, error) {
+	teamFlag := string(flag)
+	team := target.Team()
+	var err error
+	if teamFlag != "" {
```

Comment:
The `Name()` func above does the casting for you.
```suggestion
	team := target.Team()
	var err error
	if flag.Name() != "" {
```